www/page_data.inc

```diff
@@ -476,6 +476,12 @@ function loadPageStepData($localPaths, $runCompleted, $testInfo = null) {
           }
         }
       }
+
+      foreach (['firstPaintedHero', 'LastPaintedHero'] as $metric) {

```

Comment:
FirstPaintedHero Needs to be capitalized.

Reply:
Ah gee, thanks! Shoddy testing on my behalf.